spec/features/donuts/01_user_can_view_a_list_of_donuts_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+feature "user sees a list of donuts", %{
+   As a user
+   I want to be able to navigate to the donuts' page
+   To look at delicious donuts
+  } do
+  # ACCEPTANCE CRITERIA:
+  # Root is Donut#index
+  # See Links on Donut names
+
+  scenario "sees a list of donuts and a link for a new donut" do
+    @donut = create(:donut)
```

Comment:
Since this happens the same way for each test in this file, bring it out into a let statement at the beginning of the feature spec